python3-aliyun-img-utils.spec

```diff
@@ -27,6 +27,7 @@ License:        GPL-3.0-or-later
 Group:          Development/Languages/Python
 URL:            https://github.com/SUSE-Enceladus/aliyun-img-utils
 Source:         https://files.pythonhosted.org/packages/source/a/aliyun-img-utils/aliyun-img-utils-%{version}.tar.gz
+BuildRequires:  %{python_module rpm-macros}
```

Comment:
Use `python-rpm-macros`. `python_module` may expand to `python38` in places where multi Python build is set up and that does not exist. There is a `Provides: python3-rpm-macros` but nothing else.

Reply:
Rebased